tests/integration/project/elements/sandbox-bwrap/script-dev-shm-not-mounted.bst

```diff
@@ -0,0 +1,11 @@
+kind: script
+
+build-depends:
+- base.bst
+
+config:
+  create-dev-shm: false
+
+  commands:
+  - |
+    mountpoint -q /dev/shm || true
```

Comment:
Same here, this will always succeed. To ensure it's not mounted we need something like:
```! mountpoint -q /dev/shm```

Reply:
Done